.ci-self/esy-build-steps.yml

```diff
@@ -5,21 +5,21 @@ steps:
   - template: utils/use-esy.yml
   - script: "npm install -g pesy@next"
     displayName: "install pesy" 
-  - script: "pesy --fetch-cache=false --directory $(Pipeline.Workspace)/hello-reason --template=reason-native-web/morph-graphql-server-pesy-template#$(Build.SourceBranch)"
+  - script: "pesy --fetch-cache=false --directory $(Agent.HomeDirectory)/hello-reason --template=reason-native-web/morph-graphql-server-pesy-template#$(Build.SourceBranch)"
```

Comment:
This ensured teh working directory is on the same drive (C:\) as the one which has esy store